xml/System.Threading/Interlocked.xml

```diff
@@ -49,6 +49,8 @@
 3.  Store the value in the instance variable.  
   
  If you do not use <xref:System.Threading.Interlocked.Increment%2A> and <xref:System.Threading.Interlocked.Decrement%2A>, a thread can be preempted after executing the first two steps. Another thread can then execute all three steps. When the first thread resumes execution, it overwrites the value in the instance variable, and the effect of the increment or decrement performed by the second thread is lost.  
+
+ The <xref:System.Threading.Interlocked.Add%2A> method adds an integer value to an integer variable and returns the new value of the variable.  
```

Comment:
I think we need to add an "atomically" here -- either "atomically adds" or "new value of the variable as an atomic operation."

Reply:
@rpetrusha I agree; fixed!